GliaWidgets/Sources/EntryWidget/EntryWidgetView.swift

```diff
@@ -56,7 +55,9 @@ private extension EntryWidgetView {
                 isPlaceholder: true
             )
             .redacted(reason: .placeholder)
-            .disabled(true)
+            .accessibility(hidden: false)
+            .accessibilityElement(children: .combine)
+            .accessibilityLabel(Localization.EntryWidget.Loading.Accessibility.label)
```

Comment:
To keep consistency we need to keep Accessibility values as a part of Style

Reply:
@EgorovEI Good catch.